packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -135,110 +138,123 @@ describe('plugin-conversation', () => {
       it('should convert a "us" cluster to WEBEX_CONVERSATION_DEFAULT_CLUSTER cluster', async () => {
         await webex.internal.conversation.getUrlFromClusterId({cluster: 'us'});
 
-        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {clusterId: process.env.WEBEX_CONVERSATION_DEFAULT_CLUSTER});
+        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {
+          clusterId: process.env.WEBEX_CONVERSATION_DEFAULT_CLUSTER,
+        });
       });
 
       it('should add the cluster service when missing', async () => {
         await webex.internal.conversation.getUrlFromClusterId({cluster: 'urn:TEAM:us-west-2_r'});
 
-        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {clusterId: 'urn:TEAM:us-west-2_r:identityLookup'});
+        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {
+          clusterId: 'urn:TEAM:us-west-2_r:identityLookup',
+        });
       });
     });
 
     describe('paginate', () => {
       it('should throw an error if a page is passed with no links', () => {
-        try {
-          webex.internal.conversation.paginate({page: {}});
-        }
-        catch (error) {
+        webex.internal.conversation.paginate({page: {}}).catch((error) => {
```

Comment:
We should add a `fail()` call after the call to paginate() so that the test fails if paginate doesn't throw. Or even better we should just use `toThrow()` to check that a function throws.